### PR TITLE
Various grammatical changes, spelling corrections, formatting standardizations

### PR DESCRIPTION
**Pull request by ToxiClay**
#### Summary

SUMMARY: None

#### Purpose of change

The purpose of this change is to introduce several grammatical changes, as well as spelling corrections and formatting standardizations, to several item description JSON files.

#### Describe the solution

I corrected/reorganized multiple item descriptions across several JSON files within the codebase.

#### Describe alternatives you've considered

I could have left them all alone.

#### Testing

These changes do not affect actual game processing; therefore, no testing is required.

### CODE REVIEW

File: data/json/snippets/snippets.json [hunk 1]
```diff
@@ -43,7 +43,10 @@
     "type": "snippet",
     "category": "charsheet",
     "text": [
-      { "id": "charsheet_1", "text": "It's a Dungeons & Dragons 6th Edition character sheet.  This one is for a fighter." },
```
Comment (Maleclypse) [inline data/json/snippets/snippets.json:46]: This is dictated by our JSON dialect.  Changing from line 46 would permanently break our json validation tests.

**Review by Maleclypse — CHANGES_REQUESTED** [review]
There may be other json validation requirements that I've missed but those can all be found  and fixed by linting your changes.

File: data/json/snippets/snippets.json (hunk 1 above)
Comment (ToxiClay) [reply data/json/snippets/snippets.json:46]: Hrm. I apologize for that; my rationale was that I was standardizing with the lines below it.

Comment (ToxiClay) [reply data/json/snippets/snippets.json:46]: Uhh, how do I accept this change? Last time there was a button for it. Do I have to manually commit this time? Sorry, relatively new.

Comment (Maleclypse) [reply data/json/snippets/snippets.json:46]: https://dev.narc.ro/cataclysm/format.html

Yeah those two lines look very similar so they are probably a few characters off from each other and one tripped the character limit and the other didn't.  No problem, once you run your files through the linter and copy them back into your editor then reupload them to github I'll rereview.

Comment (ToxiClay) [reply data/json/snippets/snippets.json]: OK, I've re-pushed the files that failed linting. What's really weird is that some lines are fine, and some aren't. You said there's a character limitation per line that causes linting to fail? I'll have to bear this linter in mind going forward; thanks!

File: data/json/monsters/zed_misc.json [hunk 2]
```diff
@@ -1083,7 +1083,7 @@
     "id": "mon_zombie_smoker",
     "type": "MONSTER",
     "name": { "str": "smoker zombie" },
-    "description": "A blackened and twisted naked human body, strips of flesh hang from its body, and it emits a constant haze of thick black smoke.",
+    "description": "A blackened and twisted naked human body. Strips of flesh hang from its frame, and it emits a constant haze of thick black smoke.",
```
Comment (Maleclypse) [inline data/json/monsters/zed_misc.json:1086]:
```suggestion
    "description": "A blackened and twisted naked human body.  Strips of flesh hang from its frame, and it emits a constant haze of thick black smoke.",
```
Two spaces after period inside of " "

**Review by Maleclypse — APPROVED** [review]

**Pull request merged**